prime-angular-frontend/src/app/app.routes.ts

```diff
@@ -5,4 +5,5 @@ export class AppRoutes {
   public static PAGE_NOT_FOUND = 'page-not-found';
   public static HELP = 'help';
   public static DEFAULT = '**';
+  public static ACCESS_DECLINED = 'access-declined';
```

Comment:
Feels like this is left over from your refactor, and now exists in the EnrolmentRoutes